frontend/lib/components/utils/task_tree.dart

```diff
@@ -0,0 +1,168 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+import 'package:frontend/utils/task.dart';
+
+import 'package:frontend/components/utils/createTask/if_this_button.dart';
+import 'package:frontend/components/utils/createTask/then_that_button.dart';
+
+class TaskTree extends StatelessWidget {
+  const TaskTree({
+    required this.task,
+    Key? key,
+  }) : super(key: key);
+
+  final Task task;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        if (task.author != null)
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              const Text(
+                'Créée par ',
+                style: TextStyle(
+                  fontWeight: FontWeight.w500,
+                  fontSize: 14,
+                ),
+              ),
+              Text(
+                '${task.author}',
+                style: const TextStyle(
+                  fontSize: 14,
+                  fontWeight: FontWeight.w700,
+                ),
+              ),
+            ],
+          ),
+        if (task.author != null) const SizedBox(height: 15),
+        if (task.action == null) const IfThisButton(),
+        if (task.action != null)
+          _TaskServiceViewer(
+            isAction: true,
+            color: task.action!.service.color,
+            title: task.action!.name,
+            iconPath: task.action!.service.iconPath,
+            onClick: () {
+              // Get.to(
+              //   EditTask(task: task),
+              //   transition:
+              //       kIsWeb ? Transition.noTransition : Transition.rightToLeft,
+              // );
```

Comment:
?

Reply:
fixed in [6c55f91](https://github.com/Cleopha/ifttt-like/pull/66/commits/6c55f914bbd793c6159f804eafd9b291aaed715b)